src/org/opensolaris/opengrok/history/CVSRepository.java

```diff
@@ -67,7 +67,7 @@
             = Pattern.compile("([\\.\\d]+)\\W+\\((\\w+)");
 
     public CVSRepository() {
-        working = Boolean.FALSE;
+        working = null;
```

Comment:
Yes because its anchestor sets it to TRUE. See [my comment below](https://github.com/OpenGrok/OpenGrok/pull/1473#issuecomment-287709907)